src/components/table/table-row/table-row.js

```diff
@@ -170,37 +196,37 @@ class TableRow extends React.Component {
   }
 
   /**
-   * @method componentWillUnmount
-   * @return {Void}
-   */
-  componentWillUnmount() {
-    if (this.context.detachFromTable) {
-      this.context.detachFromTable(this.rowID);
-    }
-  }
-
-  /**
    * @method componentWillReceiveProps
    * @return {Void}
    */
   componentWillReceiveProps(nextProps) {
-    if (this.props.uniqueID != nextProps.uniqueID) {
+    if (this.props.uniqueID !== nextProps.uniqueID) {
```

Comment:
uniqueID doesn't seem to be called out in prop types. If we set this to only strings we could ensure that this check will always work.

Alternatively we could .toString each side. Or we can get rid of the rule but still prefer triple equality.

Problem is that i would comment if seen that a new PR didn't have a triple equals. the linter would catch this beforehand